modules/installation-configuration-parameters.adoc

```diff
@@ -1391,7 +1460,7 @@ in vSphere.
 |String, for example, `/<datacenter_name>/vm/<folder_name>/<subfolder_name>`.
 
 |`platform.vsphere.resourcePool`
-|Optional. The absolute path of an existing resource pool where the installer creates the virtual machines. If you do not specify a value, resources are installed in the root of the cluster `/<datacenter_name>/host/<cluster_name>/Resources`.
+|Optional. The absolute path of an existing resource pool where the installation program creates the virtual machines. If you do not specify a value, the installation program will install the resources in the root of the cluster `/<datacenter_name>/host/<cluster_name>/Resources`.
```

Comment:
Present tense rewrite as food for thought:

```suggestion
|Optional. The absolute path of an existing resource pool where the installation program creates the virtual machines. If you do not specify a value, the installation program installs the resources in the root of the cluster under `/<datacenter_name>/host/<cluster_name>/Resources`.
```

Reply:
Good catch, thanks